lib/mv_opentelemetry/oban.ex

```diff
@@ -0,0 +1,97 @@
+defmodule MvOpentelemetry.Oban do
+  @moduledoc false
+
+  alias OpenTelemetry.Span
+  require OpenTelemetry.SemanticConventions.Trace, as: Trace
+
+  use MvOpentelemetry.SpanTracer,
+    name: :oban,
+    events: [
+      [:oban, :job, :start],
+      [:oban, :job, :stop],
+      [:oban, :job, :exception]
+    ]
+
+  def register_tracer(opts) do
```

Comment:
🤔 Whats the difference between this and the default implementation from macro?

Reply:
There was originally a difference, but now there isn't, let me just remove it.